includes/admin/meta-boxes/class-wc-meta-box-order-actions.php

```diff
@@ -153,4 +146,32 @@ public static function save( $post_id, $post ) {
 	public static function set_email_sent_message( $location ) {
 		return add_query_arg( 'message', 11, $location );
 	}
+
+	/**
+	 * Get the available order actions for a given order.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param WC_Order|null $order The order object or null if no order is available.
+	 *
+	 * @return array
+	 */
+	public static function get_available_order_actions_for_order( $order ) {
+		$actions = array(
+			'send_order_details'              => __( 'Email invoice / order details to customer', 'woocommerce' ),
+			'send_order_details_admin'        => __( 'Resend new order notification', 'woocommerce' ),
+			'regenerate_download_permissions' => __( 'Regenerate download permissions', 'woocommerce' ),
+		);
+
+		/**
+		 * Filter: woocommerce_order_actions
+		 * Allows filtering of the available order actions for an order.
+		 *
+		 * @since x.x.x The $order param was added.
```

Comment:
Maybe do some archaeology and find out when the hook was added, so this first `@since` doesn't confuse people (i.e. the hook is not new).

Reply:
Good idea! Looks like this filter is extremely old. It existed when the file was first added in v2.1.0 in 2014 so I've added `@since 2.1.0` to match the file.

Reply:
Will be extremely interesting if it predates that version and the inaccuracy of this causes problems for someone 😁